robot.py

```diff
@@ -146,8 +146,8 @@ def teleopPeriodic(self):
         joystick_vz = -rescale_js(self.joystick.getZ(), deadzone=0.2, exponential=20.0, rate=self.spin_rate)
         joystick_hat = self.joystick.getPOV()
 
-        gamepad_vx = -rescale_js(self.gamepad.getY(10), deadzone=0.1, exponential=1.5, rate=0.5)
-        gamepad_vy = -rescale_js(self.gamepad.getX(10), deadzone=0.1, exponential=1.5, rate=0.5)
+        gamepad_vx = -rescale_js(self.gamepad.getY(wpilib.XboxController.Button.kStickRight), deadzone=0.1, exponential=1.5, rate=0.5)
+        gamepad_vy = -rescale_js(self.gamepad.getX(wpilib.XboxController.Button.kStickRight), deadzone=0.1, exponential=1.5, rate=0.5)
```

Comment:
I think you'll find this enum doesn't match either.

Reply:
Why isn't it?

Reply:
https://robotpy.readthedocs.io/projects/wpilib/en/stable/wpilib/XboxController.html#wpilib.xboxcontroller.XboxController.getX

Reply:
Ohh I see :eyeglasses: